### ISSUE TITLE
Worker not available

### ISSUE THREAD

**Issue by sgriosadair** [BananoCoin/boompow-legacy#15]
```
CRITICAL: Worker not available at http://127.0.0.1:7000/
Client not connected, waiting for it
```
Getting this error every time when starting the bat file. I've let it sit for hours and it never changes.
Windows 10.

**Comment by sgriosadair** [comment]
Doohhh... so this took a while to figure out. But apparently the GPU identifiers aren't what I thought they would be.. my system shows PCI Bus 11, device 0... for example and you'd think that would mean --gpu 11:0.. but it isn't that or 0:11.. its just a straight count from 0:0 up for each GPU apparently... and now it works.

**Comment by steezeburger** [comment]
I'm trying to run on an older 2015 macbook pro:

using any number with `-c` or using `-g 0:0`, I still always get this 

```
CRITICAL: Worker not available at http://127.0.0.1:7000/
Client not connected, waiting for it
```

but the nano work server does in fact appear to be running and ready to receive requests. Here is the output from
`$ ./bin/macos/nano-work-server -g 0:0`
```
Initializing GPU: Intel Intel(R) Core(TM) i7-4770HQ CPU @ 2.20GHz
Configured for the live network with threshold fffffff800000000
Ready to receive request on [::1]:7076
```

Any ideas?
